src/main/java/io/xdb/core/command/TimerCommand.java

```diff
@@ -0,0 +1,13 @@
+package io.xdb.core.command;
+
+import lombok.Getter;
+
+@Getter
+public class TimerCommand implements BaseCommand {
+
+    private long delayInSeconds;
+
+    public TimerCommand(final long delayInSeconds) {
```

Comment:
I would prefer to keep the way of iWF, so that the user code will look more clear:
```java
TimerCommand.byDuration( Duration.OfHour(10));
```
compared to 
```java
new TimerCommand( 10 * 3600);
```

Reply:
If using `TimerCommand.byFireTime`, how are we going to deal with the case where server and client have different time or different timezone?

Reply:
It's just local helper. You will write the code to convert this FireTime into DelayInSeconds (compared to current time)